### ISSUE TITLE
One of our beret's textures doesn't match its UV map.

### ISSUE THREAD

**Issue by Lupus590** [34th-Para-Recon-Brimstone-Company/34th-PRC-Aux-Mod#6]
**Describe the bug**
One of our beret's textures doesn't match its UV map.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to ace arsenal
2. find and equip one of our beret's (the one with our logo as large without the patch thing)
3. Look closely at the patch

**Expected behaviour**
It looks good.

**Screenshots**
TODO, screenshots.

**Additional context**
I think we'll need to learn how to do our own materials and stuff.

**Comment by Lexicon-Prime** [comment]
Beret 1
![Image](https://github.com/34th-Para-Recon-Brimstone-Company/34th-PRC-Aux-Mod/assets/131489713/18d698c8-2613-40e1-93d2-8f77e43ca148)
Beret 2
![Image](https://github.com/34th-Para-Recon-Brimstone-Company/34th-PRC-Aux-Mod/assets/131489713/efa80b5c-4db7-46fc-b306-7601c85a75d9)

Beret 2 additionally has better texturing for the rest of the hat, ignoring the patch. Once a new unit logo is chosen, I will work on this.

**Comment by Lupus590** [comment]
[beret_01.zip](https://github.com/34th-Para-Recon-Brimstone-Company/34th-PRC-Aux-Mod/files/14317315/beret_01.zip)

The original file that I used for making the textures. I used Krita to make this so if you have problems opening it then you may want to try using Krita.

**Comment by RoutOfWolves** [comment]
Is this issue still relevant?

**Comment by Lupus590** [comment]
Yes, however the files that I sent in a comment may be out of date. I can send new ones if asked.
